Jenkinsfile

```diff
@@ -951,8 +987,162 @@ pipeline {
+                    }
+                    //post {
+                    //    always {
+                    //        // Dismount is currently failing.
+                    //        sh '''. ./.build_vars.sh
+                    //              DAOS_BASE=${SL_PREFIX%/install*}
+                    //              NODE=${NODELIST%%,*}
+                    //              . ./.build_vars.sh
+                    //              DAOS_BASE=${SL_PREFIX%/install*}
+                    //              clush -B -S -o '-i ci_key' -l jenkins -w ${tnodes} \
+                    //                "if ! sudo umount ${DAOS_BASE}; then echo \"Failed to unmount $DAOS_BASE\"; ps axf; fi"
+                    //              '''
+                    //    }
+                    //}
+                } // stage('Functional_Hardware_setup 1') {
+                stage('Functional_Hardware_setup 2') {
+                    when {
+                        beforeAgent true
+                        allOf {
+                            expression { env.DAOS_STACK_CI_HARDWARE_SETUP != 'true' }
+                            expression {
+                              ! commitPragma(pragma: 'Skip-func-hw-wetup').contains('true')
+                            }
+                        }
+                    }
+                    agent {
+                        label 'stage_nvme4'
+                    }
+                    steps {
+                        provisionNodes NODELIST: env.NODELIST,
+                                       node_count: 4,
+                                       profile: 'daos_ci',
+                                       inst_repos: el7_daos_repos,
+                                       inst_rpms: 'daos-' + daos_packages_version +
+                                                  ' daos-client-' + daos_packages_version +
+                                                  ' cart-' + env.CART_COMMIT + ' ' +
+                                                  el7_functional_rpms
+                        runTest stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
+                                script: '''tnodes=$(echo $NODELIST | cut -d ',' -f 1-4)
+                                           # set DAOS_TARGET_OVERSUBSCRIBE env here
+                                           export DAOS_TARGET_OVERSUBSCRIBE=1
+                                           . ./.build_vars.sh
+                                           DAOS_BASE=${SL_PREFIX%/install*}
+                                           clush -B -S -o '-i ci_key' -l jenkins -w ${tnodes} \
+                                               "sudo mkdir -p $DAOS_BASE;sudo mount -t nfs $HOSTNAME:$PWD $DAOS_BASE"
+
+                                           clush -B -S -o '-i ci_key' -l jenkins -w ${tnodes} \
+                                             "if [ ! -e /dev/pmem0 ]; then sudo ${DAOS_BASE}/install/bin/daos_server storage prepare --scm-only --force; fi"''',
```

Comment:
Same as above about being able to combine into a single `clush` command.

Strangely the `mount` in this stage in your most recent run failed:
```
[2020-02-27T00:24:03.685Z] ---------------
[2020-02-27T00:24:03.685Z] wolf-134
[2020-02-27T00:24:03.685Z] ---------------
[2020-02-27T00:24:03.685Z] mount.nfs: /var/lib/jenkins/jenkins-1/docker_1/workspace/daos-stack_daos_PR-1679 is busy or already mounted
```

which is strange because the nodes were supposedly freshly rebooted by the `provisionNodes` and the steps in the run look like it was.  So why and how did `wolf-134:$DAOS_BASE` get mounted?

Hrm.  `wolf-134` doesn't return any connection errors at all in the previous _Waiting for nodes to be ready_ step where the other nodes all returned connection failures a number of time.  I suspect `wolf-134` didn't actually reboot.

`wolf-134` errored in the _Restore snapshot daos_ci-el7_ step:

```
+ cp /mnt/os_0//etc/sysconfig/network-scripts/route-ib1 /mnt/sda5/etc/sysconfig/network-scripts/route-ib1
cp: cannot stat <E2><80><98>/mnt/os_0//etc/sysconfig/network-scripts/route-ib1<E2><80><99>: No such file or directory
```

Is that also possibly the cause of your intermittent issue with an IB interface not being configured on boot?

The `wolf-134` snapshot restore seems to have done something slightly different than the other 3 nodes.  It has an extra:

```
+ mount /dev/sda5 /mnt/sda5
+ tar --strip-components=2 -xpf /mnt/os_0/base_os/base-daos_ci-el7-x86_64.tar.gz -C /mnt/sda5
```

before the `found daos_ci-el7`